plugins/course-apps/calculator/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "@openedx-plugins/course-app-calculator",
+    "version": "0.1.0",
+    "description": "Calculator configuration for courses using it",
+    "dependencies": {
+    },
+    "devDependencies": {}
+}
```

Comment:
In case it's not clear, generally each plugin is just three files:
* `Settings.jsx` which defines the UI (mostly unchanged from how it was before; just updated import paths)
* `messages.js` which contains the strings (unchanged)
* `package.json` which defines the plugin as an installable module (new)